### PR TITLE
Add scroll for filters in results page

### PR DESCRIPTION
**Pull request by Fahme**
## Fixes
Fixes https://github.com/creativecommons/cccatalog-frontend/issues/689 by @ManishBasavalinga and https://github.com/creativecommons/cccatalog-frontend/issues/904 by @abhisheknaiidu 

## Description
Add scrollbar to the filters in results page. 

## Technical details
Previously two pr's where open to tackle this problem https://github.com/creativecommons/cccatalog-frontend/pull/903 and https://github.com/creativecommons/cccatalog-frontend/pull/763

however the implementation wasn't complete so this PR is to address solving the problems described. 

- Works great on mobile device by adding a media query to only allow scrolling on desktops
- Doesn't use a package to add scrollable bar to filters list

## Tests

Currently tested manually :

- Desktop Macbook air 13 inch
- For mobile device to check responsiveness.

## Screenshots
Screen Shot 2020-05-07 at 4 25 17 AM

## Developer Certificate of Origin

<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>

### CODE REVIEW

File: src/components/Filters/SearchGridFilter.vue [hunk 1]
```diff
@@ -144,15 +144,19 @@ export default {
...
   top: 0;
   position: sticky;
+  height: auto;
 
   label {
     color: #333333;
   }
 
   &__visible {
+    @media (min-width: 768px) {
```
Comment (abhisheknaiidu) [inline src/components/Filters/SearchGridFilter.vue:156]: Could you use Bulma's [device](https://github.com/jgthms/bulma/blob/master/sass/utilities/mixins.sass#L81) mixins here? In this way, we will have consistency everywhere! you rever to this PR #881 for some help!

**Comment by brenoferreira** [comment]
@Fahme can you fix the conflicts here?

**Comment by Fahme** [comment]
@brenoferreira done

File: src/components/Filters/SearchGridFilter.vue [hunk 2]
```diff
@@ -132,6 +132,10 @@ export default {
     color: #333333;
   }
   &__visible {
+    @include desktop {
```
Comment (brenoferreira) [inline src/components/Filters/SearchGridFilter.vue:135]: You're missing `@import "node_modules/bulma/sass/utilities/_all";` in the beginning of the styles

**Comment by Fahme** [comment]
Thanks for the feedback, I didn't think much on the corner cases as i just
went with what was missing in previous PR's,  if you have a better idea on
fixing it i think it's good to close this pr 😊

**Pull request closed without merging**